tests/load/README.md

```diff
@@ -0,0 +1,30 @@
+## Summary
```

Comment:
Very nice writeup :heart:

Especially useful for tool like this that will only be used periodically.